torcheeg/datasets/module/motor_imagery/strokes.py

```diff
@@ -0,0 +1,313 @@
+import os
+from typing import Any, Callable, Dict, Tuple, Union
+from ..base_dataset import BaseDataset
+from ....utils import get_random_dir_path
+from scipy.io import loadmat
+import re
+import pandas as pd
+import mne
+
+
+class StrokePatientsMIDataset(BaseDataset):
```

Comment:
Define two datasets: one is StrokePatientsMIDataset, and the other is StrokePatientsMIProcessedDataset. This helps with unit testing and distinguishing different hook behaviors.